pandas/tests/arrays/test_integer.py

```diff
@@ -710,6 +710,21 @@ def test_to_integer_array_float():
     assert result.dtype == Int64Dtype()
 
 
+@pytest.mark.parametrize(
+    "values, dtype",
+    [
+        ([9999999999999999, 123123123123123123, 10000000000000543, np.nan], "Int64"),
+        (np.array([9999999999999999, 123123123123123123, 10000000000000543],
+         dtype=np.int64), "Int64"),
+    ],
+)
+def test_to_integer_array_large_numbers(values, dtype):
```

Comment:
can you add the issue number as a comment.

Reply:
Okay I'll add that in my next commit.

Reply:
I have added that.